neptune/internal/mockpoolfns.go

```diff
@@ -0,0 +1,32 @@
+package internal
```

Comment:
Renaming to `mockpoolutils.go` because calling in `xxx_test.go` causes the compiler to ignore it, even in the context of `go test ./...`. (Which, prima facie, is different from what the go documentation says?)